### DISCUSSION TITLE
请求增加支持联通云盘

### DISCUSSION THREAD

**Discussion by newens** [AlistGo/alist#4627] [Ideas]
大神，能否增加支持联通云盘？现在的联通云盘真心不错，价格不高，空间够大，云硬盘功能很强。
移动、电信的都支持了，唯独缺失了联通，也难怪联通原来的沃盘确实太差。
https://pan.wo.cn/

**Comment by xhofe** [comment]
https://github.com/alist-org/alist/releases/tag/v3.19.0

### &nbsp;&nbsp;&nbsp;🚨 Breaking Changes

- Listen to both http & https &nbsp;-&nbsp; by @BoYanZh and @Xhofe in https://github.com/alist-org/alist/issues/4536 [<samp>(f646d)</samp>](https://github.com/alist-org/alist/commit/f646d2a)

### &nbsp;&nbsp;&nbsp;🚀 Features

- Add OIDC single sign-on &nbsp;-&nbsp; by @xzzpig in https://github.com/alist-org/alist/issues/4496 [<samp>(881d6)</samp>](https://github.com/alist-org/alist/commit/881d6e2)
- Support delayed start &nbsp;-&nbsp; by @Xhofe in https://github.com/alist-org/alist/issues/4532 [<samp>(f9e06)</samp>](https://github.com/alist-org/alist/commit/f9e067a)
- Set `ProxyFromEnvironment` for default http client &nbsp;-&nbsp; by @hunshcn in https://github.com/alist-org/alist/issues/4546 [<samp>(47976)</samp>](https://github.com/alist-org/alist/commit/47976af)
- *Add `WoPan` driver &nbsp;-&nbsp; by @Xhofe in https://github.com/alist-org/alist/issues/4541 [<samp>(f7bae)</samp>](https://github.com/alist-org/alist/commit/f7baec2)*
- **139**: Auto extract account from `Authorization` &nbsp;-&nbsp; by @Xhofe [<samp>(a66b0)</samp>](https://github.com/alist-org/alist/commit/a66b0e0)
- **baidu_netdisk**: Allow custom crack ua &nbsp;-&nbsp; by @Xhofe [<samp>(1e626)</samp>](https://github.com/alist-org/alist/commit/1e62666)

### &nbsp;&nbsp;&nbsp;🐞 Bug Fixes

- **139**:
  - Modify the authentication mode &nbsp;-&nbsp; by @Xhofe [<samp>(6cbd2)</samp>](https://github.com/alist-org/alist/commit/6cbd253)
  - Avoid panic due to `Authorization` for emptiness &nbsp;-&nbsp; by @Xhofe [<samp>(e23f0)</samp>](https://github.com/alist-org/alist/commit/e23f00f)
- **189pc**:
  - Add `REQID` header &nbsp;-&nbsp; by @Xhofe [<samp>(3bfa0)</samp>](https://github.com/alist-org/alist/commit/3bfa00d)
- **aliyundrive_open**:
  - `openFile/list` rate limit &nbsp;-&nbsp; by @Xhofe [<samp>(ca5a5)</samp>](https://github.com/alist-org/alist/commit/ca5a53f)
  - Change default `oauth_token_url` &nbsp;-&nbsp; by @Xhofe [<samp>(6cd81)</samp>](https://github.com/alist-org/alist/commit/6cd8151)
- **deps**:
  - Update module github.com/gin-gonic/gin to v1.9.1 [security] [skip ci] &nbsp;-&nbsp; by @renovate[bot] in https://github.com/alist-org/alist/issues/4521 [<samp>(5141b)</samp>](https://github.com/alist-org/alist/commit/5141b3c)
- **s3**:
  - Optional add filename to disposition &nbsp;-&nbsp; by @Xhofe in https://github.com/alist-org/alist/issues/4538 [<samp>(4dca5)</samp>](https://github.com/alist-org/alist/commit/4dca52b)

### &nbsp;&nbsp;&nbsp;🏎 Performance

- **baidu**: Avoid refreshing the token on every startup &nbsp;-&nbsp; by @Xhofe [<samp>(b124f)</samp>](https://github.com/alist-org/alist/commit/b124fdc)

##### &nbsp;&nbsp;&nbsp;&nbsp;[View changes on GitHub](https://github.com/alist-org/alist/compare/v3.18.0...v3.19.0)
